src/packages/cli/bin/gw-types.js

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env node
+/* eslint-disable @typescript-eslint/no-var-requires */
+var import_graphweaver_builder = require('@exogee/graphweaver-builder');
```

Comment:
Why is this file js

Reply:
The js file is a bin file and is executed in a mac or windows env. It is a `js` file to avoid any build step and follows the same pattern as the `gw-bin.js` file.